src/Sarif/SarifUtilities.cs

```diff
@@ -184,7 +184,7 @@ public static int GetByteLength(string s, Encoding encoding)
         public static Regex RegexFromPattern(string pattern) =>
             new Regex(
                 pattern,
-                RegexOptions.Compiled | RegexOptions.CultureInvariant | RegexOptions.ExplicitCapture | RegexOptions.IgnorePatternWhitespace);
+                RegexOptions.Compiled | RegexOptions.CultureInvariant | RegexOptions.IgnorePatternWhitespace);
```

Comment:
>IgnorePatternWhitespace [](start = 85, length = 23)

Explicit capture is a more performant option for intensive regex use. I deleted it for expedience sake, but would be glad to update the regexes to use explicitly named groups. Will do this as a follow-on.